test/fixtures/reviews_approved.json

```diff
@@ -36,6 +36,15 @@
   "publishedAt": "2017-10-24T14:49:52Z"
 },
 {
+  "bodyText": "LGTM",
+  "state": "COMMENTED",
+  "author": {
+    "login": "Baz"
```

Comment:
Can you make a new case for this? This and the case above are overlapping each other so one of them would not be visible in the tests. To do this we need to add another collaborator in `collaborators.json` and the readme in the fixtures as well.

Reply:
Sure thing!

Reply:
There are in fact numerous tests using each files. I also had to add an entry in reviewers_approved.json and modify accordingly other tests. Push coming